From: Lennart Poettering <lennart@poettering.net>
Date: Mon, 5 Dec 2016 19:10:44 +0100
Subject: man: drop reference to %U being useless

This paragraph was a missed left-over from
79413b673b45adc98dfeaec882bbdda2343cb2f9. Drop it now.

(cherry picked from commit 13e40f5a4cd2cbecd3d35e0d6b277749b1d21272)
---
 man/systemd.unit.xml | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/man/systemd.unit.xml b/man/systemd.unit.xml
index 40c4cfd..a767b2e 100644
--- a/man/systemd.unit.xml
+++ b/man/systemd.unit.xml
@@ -1314,13 +1314,6 @@
       </tgroup>
     </table>
 
-    <para>Please note that specifiers <literal>%U</literal>,
-    <literal>%h</literal>, <literal>%s</literal> are mostly useless
-    when systemd is running in system mode. PID 1 cannot query the
-    user account database for information, so the specifiers only work
-    as shortcuts for things which are already specified in a different
-    way in the unit file. They are fully functional when systemd is
-    running in <option>--user</option> mode.</para>
   </refsect1>
 
   <refsect1>
